CUSTOM.CPP

```diff
@@ -1,6 +1,289 @@
 #include "all.h"
 
-char* kodtobillnev(int billkod);
+// Customize kiirashoz kell:
```

Comment:
> The function was originally in CUSTOM.CPP until it started to include windows-exclusive code (DIK codes)

What do you mean "until it started to include". Was that something we added and then removed? Or was it always there?

Reply:
That was written by sunl, I was assuming it was about Across. It seems neither `kodtobillnev` or `CUSTOM.CPP` exist in Across, so maybe it was earlier versions of Elma.